ramls/schemas/SamlRegenerateResponse.json

```diff
@@ -4,6 +4,7 @@
   "description": "Wraps sp metadata XML content (base64 encoded) in JSON.",
   "properties": {
     "fileContent": {
+      "description": "sp metadata",
```

Comment:
`sp` is usually written in capitals and should be explained, for example "SP (service provider) metadata", the same three lines before.